src/package.xml

```diff
@@ -1,2723 +1,2725 @@
-        <members>npe01__OppPayment__c.Payment_Acknowledgment_Status__c</members>
-        <members>npe03__Recurring_Donation__c.Always_Use_Last_Day_Of_Month__c</members>
-        <members>npe03__Recurring_Donation__c.Day_of_Month__c</members>
-        <members>npe03__Recurring_Donations_Settings__c.Recurring_Donation_Batch_Size__c</members>
-        <members>npe4__Relationship_Settings__c.Enable_Custom_Field_Sync__c</members>
-        <members>npe4__Relationship__c.Related_Opportunity_Contact_Role__c</members>
-        <members>npe5__Affiliation__c.Related_Opportunity_Contact_Role__c</members>
-        <members>npo02__Household__c.Number_of_Household_Members__c</members>
-        <members>npo02__Households_Settings__c.Matched_Donor_Role__c</members>
-        <members>npo02__Households_Settings__c.Use_Dated_Conversion_Rates__c</members>
-        <name>CustomField</name>
-    </types>
-    <types>
-        <members>Addr_Id_Error</members>
-        <members>Addr_No_Batch</members>
-        <members>Addr_Pending_Verification</members>
-        <members>Addr_Settings_API_Body</members>
-        <members>Addr_Settings_API_Title</members>
-        <members>Addr_Settings_Intro_Body</members>
-        <members>Addr_Settings_Intro_Body1</members>
-        <members>Addr_Settings_Intro_Body2</members>
-        <members>Addr_Settings_Intro_Title</members>
-        <members>Addr_Settings_Notice</members>
-        <members>Addr_Settings_Test_Body</members>
-        <members>Addr_Settings_Test_Response_Title</members>
-        <members>Addr_Settings_Test_Title</members>
-        <members>Addr_Skip_Verified</members>
-        <members>Addr_Token_Error</members>
-        <members>Addr_Unauthorized_Endpoint</members>
-        <members>Addr_Valid_Key_Pair</members>
-        <members>Addr_Verification_Batch_Body</members>
-        <members>Addr_Verification_Batch_Not_Supported</members>
-        <members>Addr_Verification_Batch_SmartyStreets_Message</members>
-        <members>Addr_Verification_Batch_Status</members>
-        <members>Addr_Verification_Batch_Title</members>
-        <members>Addr_Verification_Cant_Load_Class</members>
-        <members>Addr_Verification_Required</members>
-        <members>Addr_Verification_Undefined_Class</members>
-        <members>Addr_Verification_Wrong_Class</members>
-        <members>Addr_Verification_Wrong_Interface</members>
-        <members>Addr_Verified</members>
-        <members>Addr_Verify_Endpoint</members>
-        <members>Addr_Verify_Google_Help</members>
-        <members>Addr_Verify_Settings</members>
-        <members>Addr_Verifying</members>
-        <members>Address_Not_Found</members>
-        <members>Address_StateCountry_Invalid_Error</members>
-        <members>Address_Verification_Limit</members>
-        <members>CMT_DeleteConfirm</members>
-        <members>CMT_FilterGroupActiveRollupsHelpText</members>
-        <members>CMT_FilterGroupDelet
```

Comment:
Can you remove this file from the PR, with the exception of the three new classes?

Reply:
@force2b done!